README.md

```diff
@@ -20,7 +20,7 @@ Rook integrates deeply into cloud native environments leveraging extension point
 For more details about the storage solutions currently supported by Rook, please refer to the [project status section](#project-status) below.
 We plan to continue adding support for other storage systems and environments based on community demand and engagement in future releases. See our [roadmap](ROADMAP.md) for more details.
 
-Rook is hosted by the [Cloud Native Computing Foundation](https://cncf.io) (CNCF) as sandbox level project. If you are a company that wants to help shape the evolution of technologies that are container-packaged, dynamically-scheduled and microservices-oriented, consider joining the CNCF. For details about who's involved and how Rook plays a role, read the CNCF [announcement](https://www.cncf.io/blog/2018/01/29/cncf-host-rook-project-cloud-native-storage-capabilities).
+Rook is hosted by the [Cloud Native Computing Foundation](https://cncf.io) (CNCF) as incubating level project. If you are a company that wants to help shape the evolution of technologies that are container-packaged, dynamically-scheduled and microservices-oriented, consider joining the CNCF. For details about who's involved and how Rook plays a role, read the CNCF [announcement](https://www.cncf.io/blog/2018/01/29/cncf-host-rook-project-cloud-native-storage-capabilities).
```

Comment:
"as an incubating level project" sounds better to me

Reply:
fixed.